.github/workflows/rubocop.yml

```diff
@@ -0,0 +1,22 @@
+---
+name: Rubocop
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+      TESTOPTS: "-v"
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Ruby 3.2
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.2
+          bundler-cache: true
+      - name: bundle install
```

Comment:
Note that this isn't necessary, `setup-ruby` will run `bundle install` if you have `bundler-cache: true`.

Reply:
Cool will fix. I just copy-pasted from another project.